### DISCUSSION TITLE
How to wake up the reTerminal LCD after sleep | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#672] [Q&A]
# How to wake up the reTerminal LCD after sleep | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/Wakeup_reTerminal_LCD_after_sleep/

**Comment by zachdolan16** [comment]
Is there a better solution for this issue? We have had customers complain that the reTerminal display they had had a blank screen, and wouldn't wake after touching screen or pressing any of the buttons. Only cycling power to the display would fix the issue.

This "fix" where you turn on/off the display via a command is okay... but doesn't seem like a permanent fix for an "commercial" product.
